packages/vue/scripts/build-web-types.js

```diff
@@ -1,16 +1,39 @@
-const fs = require("fs")
-const docs = require("@ionic/core/dist/docs.json")
-const { pascalCase } = require('change-case')
+const fs = require("fs");
+const docs = require("@ionic/core/dist/docs.json");
+const { pascalCase } = require("change-case");
 
-const components = []
+const components = [];
 
-for (const component of docs.components) {
-  if (!component.usage.vue) continue
-  const attributes = []
-  const slots = []
-  const events = []
-  const componentName = pascalCase(component.tag)
-  const docUrl = "https://ionicframework.com/docs/api/" + component.tag.substr(4)
+/**
+ * The list of tag names to ignore generating web types for.
+ */
+const excludeComponents = [
+  "ion-app",
+  "ion-icon",
+  "ion-nav",
+  "ion-nav-link",
+  "ion-tabs",
+  "ion-tab-bar",
```

Comment:
The old web types had data for tabs and tab bar: https://cdn.jsdelivr.net/npm/@ionic/vue@6.1.0/dist/web-types.json (search for "IonTabs" or "IonTabBar"